src/InlineText.php

```diff
@@ -7,4 +7,34 @@
  */
 class InlineText extends Text
 {
+
+    public function __construct($text = "", $markdownTag = "")
+    {
+        parent::__construct($text);
+        $this->mdTag = $markdownTag;
```

Comment:
Oh yeah, I just saw you'd been adding privates to other classes after I pushed this. What's the difference here? Is it just that it's automatically public otherwise?

Reply:
@graycodes not sure, but it should be specified for readability anyway.

Reply:
k, cool